Content.Server/SurveillanceCamera/Systems/SurveillanceCameraSystem.cs

```diff
@@ -428,16 +433,11 @@ public sealed class OnSurveillanceCameraViewerRemoveEvent : EntityEventArgs
 
 }
 
-// What happens when a camera deactivates.
-public sealed class SurveillanceCameraDeactivateEvent : EntityEventArgs
-{
-    public EntityUid Camera { get; }
+[ByRefEvent]
+public record struct SurveillanceCameraActivateEvent(EntityUid Camera);
```

Comment:
you can combine these events into a single event, with the new camera status information being transmitted

Reply:
I'm always on the fence about when to do this (using a bool vs defining two events). Beyond style choices and mild logic changes, is there a reason to favor one over the other?

Reply:
Maybe not.
You can, not should